File: app/src/main/java/com/candy/caishenaccountbook/home/MainActivity.java
package com.candy.caishenaccountbook.home;

import android.os.Bundle;
import android.util.SparseArray;

import com.candy.caishenaccountbook.R;
import com.candy.caishenaccountbook.home.ui.accountbook.AccountBookFragment;
import com.candy.caishenaccountbook.home.ui.me.MeFragment;
import com.candy.caishenaccountbook.home.ui.record.RecordFragment;
import com.candy.rohmerui.RohmerBottomNavigationView;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import androidx.fragment.app.Fragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        initView();
    }

    private void initView() {
        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        RohmerBottomNavigationView navView = findViewById(R.id.nav_view);
        navView.setAdapter(createAdapter());
    }

    private RohmerBottomNavigationView.Adapter createAdapter() {
        SparseArray<Fragment> array = new SparseArray<>();
        array.put(R.id.navigation_me, new MeFragment());
        array.put(R.id.navigation_account, new AccountBookFragment());
        array.put(R.id.navigation_record, new RecordFragment());
        RohmerBottomNavigationView.Adapter adapter = new RohmerBottomNavigationView.Adapter(getSupportFragmentManager(), R.id.nav_container);
        adapter.setData(array);
        adapter.setFirstCheckItem(R.id.navigation_account);
        return adapter;
    }
}

File: app/src/main/java/com/candy/caishenaccountbook/home/MainActivity.kt
package com.candy.caishenaccountbook.home

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.util.SparseArray
import android.view.View
import androidx.fragment.app.Fragment
import com.candy.caishenaccountbook.R
import com.candy.caishenaccountbook.base.BaseActivity
import com.candy.caishenaccountbook.home.ui.accountbook.AccountBookFragment
import com.candy.caishenaccountbook.home.ui.me.MeFragment
import com.candy.caishenaccountbook.home.ui.record.RecordFragment
import com.candy.rohmerui.RohmerBottomNavigationView

class MainActivity : BaseActivity() {

    var container : View? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initView()
    }

    private fun initView() {
        val navView = findViewById<RohmerBottomNavigationView>(R.id.nav_view)
        navView.setAdapter(createAdapter())
    }

    private fun createAdapter(): RohmerBottomNavigationView.Adapter {
        val array = SparseArray<Fragment>()
        array.put(R.id.navigation_me, MeFragment())
        array.put(R.id.navigation_account, AccountBookFragment())
        array.put(R.id.navigation_record, RecordFragment())
        val adapter = RohmerBottomNavigationView.Adapter(supportFragmentManager, R.id.nav_container)
        adapter.setData(array)
        adapter.firstCheckItem = R.id.navigation_account
        return adapter
    }

    companion object {
        fun start(context : Context) {
            val intent = Intent()
            intent.setClass(context, MainActivity::class.java)
            context.startActivity(intent)
        }
    }
}